### ISSUE TITLE
AppVeyor failing after removal of bin folder.

### ISSUE THREAD

**Issue by raymelon** [raymelon/TrafficLightNeuralNetwork#5]
From https://ci.appveyor.com/project/raymelon/trafficlightneuralnetwork,

```Bash
Build started
git clone -q --branch=master https://github.com/raymelon/TrafficLightNeuralNetwork.git C:\projects\trafficlightneuralnetwork
git checkout -qf 5b15a53a564ae5ce069e88c4899ff079708d075f
Makefile.cmd
Microsoft (R) Visual C# Compiler version 1.3.1.60616
Copyright (C) Microsoft Corporation. All rights reserved.
src\NeuralNetwork.cs(25,46): warning CS0168: The variable 'ex' is declared but never used
error CS2012: Cannot open 'C:\projects\trafficlightneuralnetwork\bin\NeuralNetwork.exe' for writing -- 'Could not find a part of the path 'C:\projects\trafficlightneuralnetwork\bin\NeuralNetwork.exe'.'
Microsoft (R) Visual C# Compiler version 1.3.1.60616
Copyright (C) Microsoft Corporation. All rights reserved.
src\NeuralNetwork.cs(25,46): warning CS0168: The variable 'ex' is declared but never used
error CS2012: Cannot open 'C:\projects\trafficlightneuralnetwork\bin\TrafficLightControllerGUI.exe' for writing -- 'Could not find a part of the path 'C:\projects\trafficlightneuralnetwork\bin\TrafficLightControllerGUI.exe'.'
Command exited with code 1
```

**Comment by raymelon** [comment]
Restore bin folder, but not its .exe contents.

**Comment by raymelon** [comment]
Resolved by bfc3a2b.
